src/components/Navbar.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronLeft, faMicrophone, faCog } from '@fortawesome/free-solid-svg-icons';
+
+const Navbar = () => {
+  const navigate = useNavigate();
+  return (
+    <header>
+      <nav>
+        <div style={{ cursor: 'pointer' }} aria-hidden onClick={() => navigate(-1)}>
+          <span className="leftNav">
+            <FontAwesomeIcon icon={faChevronLeft} />
+            Africa
+          </span>
+        </div>
+        <div>
+          <span className="middleNav">Directory</span>
```

Comment:
- [ ] From the design template, **`"Directory"`** is too generic to be your caption for the app. Kindly include something more descriptive of the page contents